Doughnut/Library/Episode.swift

```diff
@@ -184,6 +184,11 @@ class Episode: Record {
     pubDate = feedItem.pubDate
     link = feedItem.link
 
+    let duration = feedItem.iTunes?.iTunesDuration
+    if (duration != nil) {
+      self.duration = Int(duration!)
```

Comment:
Force unwrapping (`!` operator) is generally considered as a code smell and should be avoided as possible. A `if let` statement can be used instead of `if (duration != nil)` to avoid the force unwrapping.